website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -166,6 +166,10 @@ These are the buckets that will be created in cumulus OU account:
 - `<PREFIX>-protected`
 - `<PREFIX>-tf-state` (for storing the terraform state file in cumulus OU account)
 
+:::tip
+The `*-tf-state` buckets and dynamoDB tables will not be automatically removed by cleanup scripts.
+Once you are done with your testing, and you have verified that cleanup is actually successful, manually delete these resources.
+:::

```

Comment:
Bucket deletion can only happen if bucket is completely empty. More importantly, a failed Terraform operation does not presently raise a clear error in Bamboo, so I'm in favor of manual checks/intervention here.